test/h2_integration/certificate_rejected_test.exs

```diff
@@ -1,9 +1,16 @@
 defmodule H2Integration.CerificateRejectedTest do
-  alias Helpers.SetupHelper, as: Tools
   use ExUnit.Case
 
+  import Mox
+  setup :set_mox_global
+  setup :verify_on_exit!
+
   alias Helpers.SetupHelper, as: Setup
 
+  setup do
+    Setup.passthrough_h2
```

Comment:
This is slightly weird, as we do make a function call without `()`, this is not recommended. We should either make a proper function call, or do it via function-less setup block:
`setup Setup.passthrough_h2`